config/rbac/harborconfiguration_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit harborconfigurations.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: harborconfiguration-editor-role
+rules:
+- apiGroups:
+  - harbor-configuration.harbor.configuration
+  resources:
+  - harborconfigurations
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - harbor-configuration.harbor.configuration
+  resources:
+  - harborconfigurations/status
+  verbs:
+  - get
```

Comment:
Why is only get needed here? Should be at least patch as well.

Reply:
this was autogenerated, we can add it now or when the status update functionality is implemented

Reply:
the harborconfiguration_editor_role.yaml file is for end users not service accounts or operators, it makes sense that the role doesn't have permission to patch or update the harborconfigurations status

Reply:
the harborconfiguration_editor_role.yaml file is for end users not service accounts or operators, it makes sense that the role doesn't have permission to patch or update the harborconfigurations status